xlators/mgmt/glusterd/src/glusterd-messages.h

```diff
@@ -322,7 +322,7 @@ GLFS_MSGID(
     GD_MSG_ADD_BRICK_MNT_INFO_FAIL, GD_MSG_GET_MNT_ENTRY_INFO_FAIL,
     GD_MSG_QUORUM_CLUSTER_COUNT_GET_FAIL, GD_MSG_POST_COMMIT_OP_FAIL,
     GD_MSG_POST_COMMIT_FROM_UUID_REJCT, GD_MSG_POST_COMMIT_REQ_SEND_FAIL,
-    GD_MSG_GRACEFUL_CLEANUP_SET_FAIL);
+    GD_PMAP_PORT_BIND_FAILED, GD_MSG_GRACEFUL_CLEANUP_SET_FAIL);
```

Comment:
Older messages cannot be modified. New messages must be added at the end of the list.

Reply:
Ohh, that happened by mistake while doing a rebase....will update that.